games/lockpicking/src/styles/pages/tabs-table.css

```diff
@@ -0,0 +1,29 @@
+.tabs-table {
```

Comment:
Refactor styles 
Not a tabs-table.
This should be or abstract or links to scores modal.
I guess this should be abstract styles. 
Something like this:
.table
.table-row
.table-head 

etc.